src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+2. The browser performs hit testing to decide which HTML frame (main frame or some iframe) an event
+   belongs to. Then the browser sends the event to the appropriate renderer process in charge of
+   that HTML frame.
+3. The renderer receives the event and queues it so that it can process when it becomes available to
+   do so.
+4. The renderer processes the event by running its handlers. These handlers may queue additional
+   asynchronous work, such as `setTimeout` and fetches, that are part of the input handling. But at
+   this point, the synchronous work is complete.
+5. A frame is painted to the screen that reflects the result of event handlers running. Note that
+   any asynchronous tasks queued by the event handlers may still be unfinished.
+
+The time between steps (1) and (3) above is an event's _delay_, which is what FID measures.
+
+The time between steps (1) and (5) above is an event's _duration_. This is what our new metric will
+measure.
+
+The event's duration includes the delay, but it also includes the work occurring in event handlers
+and the work the browser needs to do to paint the next frame after those handlers have run. The
+duration of an event is currently available in the [Event Timing API](/#event-timing-api) via the
+entry's [duration](https://w3c.github.io/performance-timeline/#dom-performanceentry-duration)
+attribute.
+
+{% Details %}
+
+{% DetailsSummary %} A note on asynchronous tasks {% endDetailsSummary %}
+
+Ideally we would love to also capture asynchronous work triggered by the event. But the problem is
+that the definition of asynchronous work triggered by the event is extremely tricky to get right. As
+an example, a developer may choose to begin some animation on event handlers and use a `setTimeout`
+to begin such animation. If we captured all tasks posted on the handlers, the animation would delay
+the completion time for as long as the animation runs. We believe it is worthwhile to investigate
+options on how to use heuristics to capture work that is asynchronous and which should be completed
+ASAP. However, we want to be really careful when doing so because we don’t want to penalize work
+that is meant to take a long time to be finished. Thus, our initial effort will look at step 5 as
+the end point: it will only consider synchronous work and the amount of time it takes to paint after
+such work is completed. That is, we’re not going to apply heuristics to guess the work that would be
+kicked off asynchronously in step 4 in our initial effort.
+
+It’s worth noting that, in many cases, work should be executed synchronously. In fact, this may be
+unavoidable because events are sometimes dispatched one after the other and the event handlers need
+to be executed in order. That said, we will still miss important work, like events which trigger
+fetching or which rely on important work to be done at the next `requestAnimationFrame` callback,
+for example. {% endDetails %}
```

Comment:
One more point: getting that sync portion+first paint to finish quickly is still uniquely important, even in situations of async follow-on work.  Typically there is *some* initial visual indication of an interaction (blue links change color, buttons look pressed, etc)